### ISSUE TITLE
Conflict with SublimeFileBrowser

### ISSUE THREAD

**Issue by akoshman** [SublimeText/WordHighlight#60]
If I set **highlight_when_selection_is_empty** option to **true** and then put cursor on any word to find its occurencies and then click to FileBrowser pane, this will highlight even matces in FileBrowser's panel:

![](http://i.imgur.com/G2BuTK5.jpg)

Normally, with option **highlight_when_selection_is_empty** turned off, it will look like this:

![](http://i.imgur.com/3WOn3y3.jpg)

Is this a conflict with SublimeFileBrowser plugin?

**Comment by adzenith** [comment]
Yeah. SublimeFileBrowser is using a normal text view to fake a file tree. This plugin has no way of knowing that it's not a normal text view like any other.
